FLT/Deformations/RepresentationTheory/Irreducible.lean

```diff
@@ -0,0 +1,34 @@
+import FLT.Deformations.RepresentationTheory.Subrepresentation
+import FLT.Mathlib.RepresentationTheory.Basic
+
+namespace Representation
+
+variable {G : Type*} [Group G]
+
+variable {k : Type*} [Field k]
+
+variable {W : Type*} [AddCommMonoid W] [Module k W]
+
+/-!
+  IsIrreducible predicates that a given Representation ρ is irreducible (also known as simple),
```

Comment:
```suggestion
  `IsIrreducible ρ` is the statement that a given representation ρ is irreducible (also known as simple),
```
My instinct is to make things easier for mathematicians to read, and I don't know if they know what "predicates" means.